docs/cli.md

```diff
@@ -0,0 +1,142 @@
+# Commandline interface
+
+The primary way we expect people to interact with `hipercow` is via the commandline interface.  This page contains the automatically generated reference documentation for this interface; longer explanations are available elsewhere in the documentation.
+
+::: mkdocs-click
+    :module: hipercow.cli
+    :command: cli
+    :depth: 1
+    :prog_name: hipercow
+    :list_subcommands: True
+    :style: table
+
+
+## Error handling
+
+If `hipercow` throws an error, by default only the actual error message is thrown and not the whole stacktrace.  Over time, we will try and make these error messages rich enough that you can debug things comfortably with what is shown on screen.
+
+```console
+$ hipercow environment new --engine other
+Error: Only the 'pip' and 'empty' engines are supported
+For more information, run with 'HIPERCOW_TRACEBACK=1'
+```
+
+To get more information you can set the `HIPERCOW_TRACEBACK` environment variable, either globally (not recommended except for developing `hipercow`) or locally:
+
+```console
+$ HIPERCOW_TRACEBACK=1 hipercow environment new --engine othe
```

Comment:
```suggestion
$ HIPERCOW_TRACEBACK=1 hipercow environment new --engine other
```

Reply:
I think this only works for bash? On windows (in hatch env) I can write

`env HIPERCOW_TRACEBACK=1 hipercow environment new --engine other`

but my command prompt is quite custom with GNU tools - I will try and confirm whether `env` works out of the box.

Reply:
we can get better windows instructions added - I should get some time with users this week too